packages/shared/src/components/Feed.tsx

```diff
@@ -338,7 +338,11 @@ export default function Feed<T>({
     nextPost: (items[postMenuIndex + 1] as PostItem)?.post,
   };
 
-  const ArticleModal = PostModalMap[selectedPost?.type];
+  const PostModal =
+    post?.sharedPost?.type === PostType.VideoYouTube &&
+    post?.source?.type === SourceType.Squad
+      ? SharePostModal
+      : PostModalMap[selectedPost?.type];
```

Comment:
Why do you need this weird evaluation?
I'm not sure why it makes a difference if the sharedPost is a video or not?

Reply:
But should it not just be a shared post then?
Nensi's PR should already reflect the rendering difference I think?

The only difference I think is the image is now a video, so it should just be another evaluation of the shared post content I personally think.

Let me know what you think, else maybe we can sync on it quickly?

Reply:
> But should it not just be a shared post then?

I was thinking of that too. I thought of checking the spec but it seemed to be empty. The way it was presented in the design looks like you should be able to play the video right away.

> Let me know what you think, else maybe we can sync on it quickly?

We can probably talk about it in our upcoming 1-1.